tests/ui/binding/underscore-prefixed-function-argument.rs

```diff
@@ -0,0 +1,15 @@
+//! Test that functions with unnamed arguments are correct handled by compiler
```

Comment:
```suggestion
//! Test that argument names starting with `_` are usable.
```

Since it's not actually unnamed